### ISSUE TITLE
Export to onnx

### ISSUE THREAD

**Issue by Z-Mooz** [gaobb/MetaUAS#7]
That's an amazing job, how to export the model to onnx for deployment? I've tried with following code, but it failed.
`# configurations
random_seed = 1
encoder_name = 'efficientnet-b4' 
decoder_name = 'unet' 
encoder_depth = 5
decoder_depth = 5
num_alignment_layers = 3
alignment_type =  'sa' 
fusion_policy = 'cat'

set_random_seed(random_seed)
metauas_model = MetaUAS(encoder_name, 
                decoder_name, 
                encoder_depth, 
                decoder_depth, 
                num_alignment_layers, 
                alignment_type, 
                fusion_policy
                )   

options = 'model-512'
if 'model-512' in options:
    ckt_path = "weights/metauas-512.ckpt"
    model = safely_load_state_dict(metauas_model, ckt_path)
    img_size = 512
else:
    ckt_path = 'weights/metauas-256.ckpt'
    model = safely_load_state_dict(metauas_model, ckt_path)
    img_size = 256
model.eval()

input_names = ['input']
ouput_names = ['mask']

prompt = torch.randn((1,3,512,512))
query = torch.randn((1,3,512,512))
input = {
        "query_image": query,
        "prompt_image": prompt,
    }

torch.onnx.export(
    model, 
    input, 
    'model-512.onnx', 
    )
`

**Comment by Z-Mooz** [comment]
It has been addressed.

**Comment by gaobb** [comment]
hi, @wawatt 

Could you please make the implementation public? Thanks.

> how to export onnx??

**Comment by wawatt** [comment]
> hi, [@wawatt](https://github.com/wawatt)
> 
> Could you please make the implementation public? Thanks.
> 
> > how to export onnx??

@Z-Mooz ^_^
